### DISCUSSION TITLE
It seem winget failed to list the manifest on local path.

### DISCUSSION THREAD

**Discussion by BrandonWanHuanSheng** [microsoft/winget-cli#3846] [Q&A] [answered]
### Description of the new feature / enhancement

I am trying to look to see if display version is correct on the manifest but somewhere when I provided to winget list -m where is the manifest directory on local path but winget report there is no matching found on package criteria which is incorrect. But the app was installed on my computer. Only can check display version after pipelines get merged.

**Comment by Trenly** [comment]
`winget list` doesn't support the `--manifest` parameter. From your write-up, it isn't clear if you're saying this is a bug, or if you're requesting that support be added for `--manifest`. Can you clarify?

```
PS C:\WINDOWS\system32> winget list -?
Windows Package Manager (Preview) v1.7.2782-preview
Copyright (c) Microsoft Corporation. All rights reserved.

The list command displays the packages installed on the system, as well as whether an upgrade is available. Additional options can be provided to filter the output, much like the search command.

usage: winget list [[-q] <query>] [<options>]

The following command aliases are available:
  ls

The following arguments are available:
  -q,--query                      The query used to search for a package

The following options are available:
  --id                            Filter results by id
  --name                          Filter results by name
  --moniker                       Filter results by moniker
  -s,--source                     Find package using the specified source
  --tag                           Filter results by tag
  --cmd,--command                 Filter results by command
  -n,--count                      Show no more than specified number of results (between 1 and 1000)
  -e,--exact                      Find package using exact match
  --scope                         Select installed package scope filter (user or machine)
  --header                        Optional Windows-Package-Manager REST source HTTP header
  --accept-source-agreements      Accept all source agreements during source operations
  --upgrade-available             Lists only packages which have an upgrade available
  -u,--unknown,--include-unknown  List packages even if their current version cannot be determined. Can only be used with the --upgrade-available argument
  --pinned,--include-pinned       List packages even if they have a pin that prevents upgrade. Can only be used with the --upgrade-available argument
  -?,--help                       Shows help about the selected command
  --wait                          Prompts the user to press any key before exiting
  --logs,--open-logs              Open the default logs location
  --verbose,--verbose-logs        Enables verbose logging for winget
  --disable-interactivity         Disable interactive prompts

More help can be found at: https://aka.ms/winget-command-list
```

**Reply by BrandonWanHuanSheng** [reply]
See here microsoft/winget-pkgs#122774. I want to check that one before validation but somewhere somehow the manifest did not list that application. I am requesting support of winget list -m

**Comment by denelon** [comment, marked as answer]
You would need to install the package to "see" what the displayVersion is.

The "displayVersion" key in the manifest should match what is displayed in Windows Apps & Features:
<img width="265" alt="displayVersion" src="https://github.com/microsoft/winget-cli/assets/61799811/c0e592f6-aea4-42f5-af27-a3cea275fa31">

I've highlighted the "displayVersion" value for Element once it's installed.

Note: Dolby Vision Extensions doesn't "show" the "displayVersion" in the top level UI.

You would need to look at "Advanced options" via the "..." widget to see the Version for packaged applications.
<img width="491" alt="PackagedDisplayVersion" src="https://github.com/microsoft/winget-cli/assets/61799811/31c563a6-93ec-4dc8-887e-22dad1c833da">

I've highlighted the "displayVersion" for Dolby Vision Extensions.

**Reply by BrandonWanHuanSheng** [reply]
See here microsoft/winget-pkgs#122774. I want to check that one before validation but somewhere somehow the manifest did not list that application. I am requesting support of winget list -m. But it does not same as that one. Actual Version: which is currently unknown because pipeline was still open Expected Behaviour same as microsoft/winget-pkgs#124544
